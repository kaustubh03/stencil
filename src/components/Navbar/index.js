// @flow
import React, { Component } from "react";
import styles from "./index.module.css";
import { Nav, NavItem } from "react-bootstrap";
type Props = {};
type State = {};
class Navbar extends Component<Props, State> {
  render() {
    return (
      <Nav bsStyle="pills" activeKey={1}>
        <NavItem className="btn" href="/crud" title="CRUD Example">
          CRUD Example
        </NavItem>
        <NavItem
          eventKey={2}
          className="btn"
          title="API Calls Using Axios"
          href="/api"
        >
          API Example
        </NavItem>
        <NavItem eventKey={3} className="btn" href="/react-icons">
          React Icons
        </NavItem>
      </Nav>
    );
  }
}
export default Navbar;
