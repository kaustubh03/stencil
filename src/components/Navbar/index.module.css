.Navbar {
  display: block;
}
a {
  background-color: #000;
  border-radius: 5px;
  color: white !important;
  padding: 0.5em !important;
  text-decoration: none !important;
  width: 78px !important;
}

a:focus,
a:hover {
  background-color: FireBrick !important;
  color: White !important;
}
