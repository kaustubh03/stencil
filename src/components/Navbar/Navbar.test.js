import React from 'react'
import renderer from 'react-test-renderer'
import Navbar from '.'

it('Navbar: default', () => {
  const component = renderer.create(<Navbar />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})