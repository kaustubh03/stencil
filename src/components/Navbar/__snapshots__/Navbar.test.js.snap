// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar: default 1`] = `
<ul
  className="nav nav-pills"
  role={null}
>
  <li
    className="btn"
    role="presentation"
  >
    <a
      href="/crud"
      onClick={[Function]}
      onKeyDown={[Function]}
      title="CRUD Example"
    >
      CRUD Example
    </a>
  </li>
  <li
    className="btn"
    role="presentation"
  >
    <a
      href="/api"
      onClick={[Function]}
      onKeyDown={[Function]}
      title="API Calls Using Axios"
    >
      API Example
    </a>
  </li>
  <li
    className="btn"
    role="presentation"
  >
    <a
      href="/react-icons"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      React Icons
    </a>
  </li>
</ul>
`;
